docs/supported-versions.md

```diff
@@ -50,12 +53,12 @@ Get acquainted with the list of the latest releases and component versions on wh
 
 |[EDP Release Version](https://github.com/epam/edp-install/blob/master/RELEASES.md)|Release Date|[EKS Version](https://aws.amazon.com/eks/)|[OpenShift Version](https://github.com/okd-project/okd/releases)|
 |:-:|:-:|:-:|:-:|
+|3.9 |Jun 13, 2024|1.29|4.12|
 |3.8 |Mar 12, 2024|1.26|4.12|
 |3.7 |Dec 15, 2023|1.26|4.12|
 |3.6 |Nov 03, 2023|1.26|4.12|
-|3.5 |Sep 21, 2023|1.26|4.12|
+|<span style="color: gray;">3.5 </span> |<span style="color: gray;">Sep 21, 2023</span>|<span style="color: gray;">1.26</span>|<span style="color: gray;">4.12</span>|
```

Comment:
Let's widen the versions lifetimes. The 3.7, 3.8, and 3.9 should the very end of the timelines, whereas 3.6 should be outlined in yellow and end in the same day when the 3.9 is released (2024-6-13).